dist/gcc-compatible/Hacl_Hash_SHA3.h

```diff
@@ -35,25 +35,38 @@ extern "C" {
 #include "krml/lowstar_endianness.h"
 #include "krml/internal/target.h"
 
-#include "Hacl_Streaming_Types.h"
+#include "Hacl_Spec.h"
```

Comment:
This is independent from this PR, but Hacl_Spec is not the greatest name for this bundle. Maybe Hacl_Hash_Types?

Reply:
there are potentially more useful types in there beyond hashes, the name (obviously) comes from the fact that this groups together all the Spec.* files... any thoughts?

Reply:
fixed in per Aymeric and I's discussion [a41202d](https://github.com/hacl-star/hacl-star/pull/803/commits/a41202dc7378b6e41c609d6b2f8f8b8e10581827)